README.rst

```diff
@@ -57,14 +53,64 @@ Usage
 Superset Assets
 ---------------
 
-OARS maintains the Superset assets in this repository, specifically the dashboards, charts, datasets, and databases. That means that any updates made here will be reflected on your Superset instance when you update your deployment.
+OARS maintains the Superset assets in this repository, specifically the dashboards,
+charts, datasets, and databases. That means that any updates made here will be reflected
+on your Superset instance when you update your deployment.
+
+But it also means that any local changes you make to these assets will be overwritten
+when you update your deployment. To prevent your local changes from being overwritten,
+please create new assets and make your changes there instead. You can copy an existing
+asset by editing the asset in Superset and selecting "Save As" to save it to a new name.
+
+Define you own superset assets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To programatically define custom superset assets there is a patch you can use with an
+inline plugin. The patch ``superset-extra-assets`` will allow you to define your
+own superset assets in a yaml file and have them automatically imported into superset
+when you run ``tutor [dev|local|k8s] init -l oars``.
+
+An example of this patch is provided as reference:
+
+..  code-block:: yaml
+
+    name: custom-inline-plugin
+    version: 0.1.0
+    patches:
+    superset-extra-assets: |
+        - file_name: my-dashboard.yaml
+            dashboard_title: "..."
+            ...
+        - file_name: my-chart.yaml
+            slice_name: "..."
+            ...
+        - file_name: my-database.yaml
+            database_name: "..."
+            ...
+        - file_name: my-dataset.yaml
+            table_name: "..."
+            ...
+
+The patch is expected to be a list of assets with an extra attribute called ``file_name`` ,
+so you can pass multiple assets separated as a **yaml** list item. Each asset is expected
+to be a valid yaml file with the attributes that superset expects for each asset type.
```

Comment:
I found this a bit confusing..

```suggestion
The patch is expected to be a list of assets with an extra attribute called ``file_name`` , which uniquely identifies the asset entry. This file does not need to exist anywhere; it will be created with the rest of the yaml in that stanza as part of the init process. Each asset is expected to be a valid yaml file with the attributes that superset expects for each asset type. See `assets.yaml <tutoroars/templates/oars/apps/superset/pythonpath/assets.yaml>`_ for examples of asset yaml declarations.
```